packages/core/src/browser/style/index.css

```diff
@@ -187,7 +187,6 @@ textarea {
 @import './menus.css';
 @import './sidepanel.css';
 @import './tabs.css';
-@import './materialcolors.css';
```

Comment:
@olexii4 Why it cannot be applied always anymore, but should be applied as part of `useable`?

Reply:
@akosyakov  Previously I added options with id='theia-theme'  to the 'style-loader/useable'  loader to have a possibility to reapply css rules into the webview(inside an iframe). Our thems depends on 'materialcolors.css' that is why we need it inside an iframe.

I try to understand the best way how to do it.  I do not want to add one more id in DOM and configure a new loader webpack or copy and add 'materialcolors.css' in each iframe as a separate resource.

I think the best way to have all variables in the separate <style> tag with id='theia-theme' and I do it.

WDYT?

Reply:
thank you for the explanation!

So if i understand it should be enough to use materialcolors.css once always not only for dark and light themes?

i.e. instead `static readonly materialColors = require('../../src/browser/style/materialcolors.css');` -> require('../../src/browser/style/materialcolors.css').use()` and better at the top of the file to make sure that it is applied before the shell is loaded?

Reply:
Thanks. I am fixed.